subsys/suit/platform/sdfw/src/suit_plat_authenticate.c

```diff
@@ -75,6 +75,16 @@ int suit_plat_authenticate_manifest(struct zcbor_string *manifest_component_id,
 		return SUIT_ERR_AUTHENTICATION;
 	}
 
+#ifdef CONFIG_ZTEST
+	if (public_key_id == MCI_KEY_ID_VOLATILE) {
```

Comment:
Actually, I think `suit_mci_signing_key_id_get` could check this under the hood, I do not thin `MCI_KEY_ID_VOLATILE` should be used here

Reply:
Changed, as suggested :slightly_smiling_face: